### ISSUE TITLE
sort tracks alphabetically when dropping folders

### ISSUE THREAD

**Issue by ciros88** [aargirakis/BZRPlayer#537]
when dropping **tracks** to tracklist/playlists windows, no sorting happens (the order comes from the source);
when droppings **folders** to tracklist/playlists windows instead, the tracklist entries seems non-alphabetically sorted (maybe natural sort it is used): alphabetical sorting should be most appropriate. here an example of a dropped folder:

- current behaviour:
![image](https://github.com/aargirakis/BZRPlayer/assets/84340161/38a93de9-881a-4219-a571-91836abf628f)

- desidered behaviour:
![image](https://github.com/aargirakis/BZRPlayer/assets/84340161/257daa2f-a9fa-4532-8be2-bd248ab46672)

example files (put all of them in the same folder):
[sorting 1.zip](https://github.com/user-attachments/files/16116656/sorting.1.zip)
[sorting 2.zip](https://github.com/user-attachments/files/16116657/sorting.2.zip)

**Comment by aargirakis** [comment]
It's based on how the filesystem sort the files. It's like "get all the files from the current folder". So if ew want another sorting we have to sort it ourself. Which will add some sort delay.

**Comment by aargirakis** [comment]
Actually you there is an option from Qt to sort the files:
https://doc.qt.io/qt-6/qdir.html#SortFlag-enum
Tried some and natural sorting doesn't seem supported though.

**Comment by ciros88** [comment]
makes sense: when dropping a folder to bzr2, if u want to sort it just click on the proper column (when this feature will be available). noticed also vlc behave as the same btw. this can be closed
